proj/src/main/webapp/static/chat.html

```diff
@@ -51,35 +59,140 @@
                                 <pre class="message-body"></pre>
                             </div>
                     </template>
-                    <button type="button" id="settings-button">settings</button>
-                </aside>
+            </div> 
+        
+            <a href="#profile-submenu" data-toggle="collapse" aria-expanded="false" class="list-group-item list-group-item-action flex-column align-items-start">
+                    <div class="d-flex w-100 justify-content-between align-items-center">
+                        <span class="fa fa-user fa-fw mr-3 fa-1x"></span>
+                        <span class="menu-collapsed"><strong>profile</strong></span>
+                        <span class="submenu-icon ml-auto"></span>
+                    </div>
+            </a>
 
+          
+            <div id='profile-submenu' class="collapse sidebar-submenu">
+            <div id="user-profile"> 
+                <div class="center-h">
+                    <div id="image-cropper">
+                        <img id="user-pfp" src="https://images.fineartamerica.com/images/artworkimages/mediumlarge/2/poppyscape-sunset-impasto-palette-knife-acrylic-painting-mona-edulesco-mona-edulesco.jpg" id="pfp" class="menu-collapsed" alt="profile picture">
```

Comment:
Maybe put some TODOs here for getting a user picture from the database; defaulting to some base picture; and getting the name, email, bio, and info from the user data.

Reply:
oh the default picture is a good idea. it pretty much load everything else from the database. there is a static profile underneath